Build-FFmpeg.ps1

```diff
@@ -303,6 +303,25 @@ if (!(Test-Path $BashExe)) {
     }
 }
 
+# Check for YASM and download if needed
+
+New-Item -ItemType Directory -Force $PSScriptRoot\Output\yasm -OutVariable yasm

```

Comment:
This is not really necessary, is it?
`yasm` can be installed in the WSL or MinGW instance via package management (tested both myself).

Reply:
Oh really? That would be even better. I will check if it works with pacman.

Reply:
Cool this works. Removed automatic yasm download and added pacman command to MSYS2 setup description.